starwars/src/components/Film.jsx

```diff
@@ -0,0 +1,42 @@
+import React,{Component} from 'react'
+
+export default class Film extends Component {
+    constructor(props){
+        super(props)
+        this.state={
+            film:null
+        }
+    }
+
+    componentDidMount(){
+        this.getFilm(this.props.filmURL)
+    }
+
+    getFilm = URL => {
+        // feel free to research what this code is doing.
+        // At a high level we are calling an API to fetch some starwars data from the open web.
+        // We then take that data and resolve it our state.
+        fetch(URL)
+          .then(res => {
+            return res.json();
+          })
+          .then(data => {
+            this.setState({film:data.title})
+          })

```

Comment:
Great job just pulling the title of the film since that's the data you wanted to display. this saves a step or 2 later